codesign/bin/verify.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Isn't the copyright 2014? Is the header check not running in the codesign directory?

Reply:
I copied the year from //codesign/bin/codesign.dart. This package seems to have its own LICENSE file also.